src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10133,9 +10195,7 @@ private bool ScanExplicitlyTypedLambda(Precedence precedence)
                             {
                                 return true;
                             }
-
                             continue;
-
```

Comment:
nit: you're removing all these blank lines.  but, IMO, these were good to have (above as well).  It allows the code to breath a bit, and not feel like very statement is jammed together ats tightly as possible :)   Just as a minor suggestion, but consider maybe not going and undoing these blank lines :)